tests/steps/cluster.py

```diff
@@ -849,6 +849,7 @@ def step_container_is_in_quorum_group(context, name):
         {
             fqdn: {
                 'state': 'streaming',
+                'sync_state': 'quorum',
```

Comment:
Is it `'quorum'` for `'sync'` replication type?
Maybe we should use `name` here?